dcos/dcos-ui
# DCOS-23825: Nodes / remove graph from nodes page (and move filter)

## ℹ️ Description

This PR removes the big graph from the Nodes Page 🎉 

- 🔖 JIRA: [DCOS-23825](https://jira.mesosphere.com/browse/DCOS-23825)
- ~⚠️Please dont merge before Lily had a chance to see it 👍~ Lily Approved ✅  

## 🚧Trade-Offs

We dont have the components and new designs for that new filter dropdown thingy _yet_, so I went ahead and simply moved the old component. Looks good on Desktop, but there is a small glitch on smaller screens (which was there before already for the list/grid switch - see first half of gif)
![screen recording 2018-07-20 at 10 19 am](https://user-images.githubusercontent.com/530632/42991642-78a127a6-8c06-11e8-831e-1fff4d3b00fe.gif)
(Gif runs in 2x speed for some reason 🤷‍♂️ )

/cc @lilysamimi 

## 💥Testing

1. Get some load on your cluster
2. Open Nodes Page and verify that Graph is removed
3. Switch between Grid & List to verify that the Filter is only displayed for Grid view
4. Change Filter in Grid view to verify that it changes content of Grid circles
